requirements-dev.txt

```diff
@@ -2,6 +2,8 @@
 pandas>=0.23
 pyarrow>=0.10,<0.11
 numpy>=1.14
+openpyxl>=2.6.2
+xlrd>=0.9.0
```

Comment:
Maybe these are under `Test` category?
And do we need to explicitly specify the version? I'm afraid it causes to break the Python 3.5 build like before (#241).

Reply:
Moved the libraries and removed the versions.